### PR TITLE
Infrastructure VM upgrade runbook

### LINKED ISSUE #1218: Mithril networks infrastructure maintenance
## Issue
Some maintenance operations need to be done on the Mithril networks.

## To do 
- [x] Add runbook for updating VMs
- [x] Cleanup old GCP bucket when empty

## Later
- Migrate new DNS zone for `api.mithril.network` in `Mithril` GCP project
- Optimize VMs (CPU, Memory, IO, ...)
- Add runbook for upgrading disks

### PR DESCRIPTION
**Pull request by jpraynaud**
## Content
This PR includes a new runbook with a detailed process to **upgrade or downgrade the VM of a Mithril network** infrastructure

## Pre-submit checklist

- Branch
  - [x] Commit sequence broadly makes sense
  - [x] Key commits have useful messages
- PR
  - [x] No clippy warnings in the CI
  - [x] Self-reviewed the diff
  - [x] Useful pull request description
  - [x] Reviewer requested
- Documentation
  - [x] Update documentation website (if relevant)

## Issue(s)
Closes #1218

### CODE REVIEW

File: docs/runbook/README.md [hunk 1]
```diff
@@ -6,15 +6,14 @@ This page gathers the available guides to operate a Mithril network.
...
+| **Genesis manually**                    | [manual-genesis](./genesis-manually/README.md)                         | Proceed to manual (re)genesis of the aggregator certificate chain.        |
+| **Era markers**                         | [era-markers](./era-markers/README.md)                                 | Create and update era markers on the Cardano chain.                       |
+| **Downloads statistics**                | [downloads statistics](./statistics/README.md)                         | Display the number of downloads per day.                                  |
+| **Signer registrations monitoring**     | [registrations-monitoring](./registrations-monitoring/README.md)       | Gather aggregated data about signer registrations (versions, stake, ...). |
+| **Update protocol parameters**          | [protocol-parameters](./protocol-parameters/README.md)                 | Update the protocol parameters of a Mithril network.                      |
+| **Recompute certificates hash**         | [recompute-certificates-hash](./recompute-certificates-hash/README.md) | Recompute the certificates has of an aggregator.                          |
```
Comment (dlachaumepalo) [inline docs/runbook/README.md:16]: This should not be:
```suggestion
| **Recompute certificates hash**         | [recompute-certificates-hash](./recompute-certificates-hash/README.md) | Recompute the certificates of an aggregator.                          |
```

**Review by dlachaumepalo — APPROVED** [review]

File: docs/runbook/README.md (hunk 1 above)
Comment (jpraynaud) [reply docs/runbook/README.md:16]: Good catch, maybe more a typo on `hash`:
```suggestion
| **Recompute certificates hash**         | [recompute-certificates-hash](./recompute-certificates-hash/README.md) | Recompute the certificates hash of an aggregator.                          |
```

Comment (jpraynaud) [reply docs/runbook/README.md]: Done :heavy_check_mark:

Comment (dlachaumepalo) [reply docs/runbook/README.md]: Yes, that's better!

**Pull request merged**